src/emulator/shader/src/translator/data.cpp

```diff
@@ -201,6 +206,11 @@ bool USSETranslatorVisitor::vmov(
     spv::Id source = load(inst.opr.src1, dest_mask, src1_repeat_offset);
     store(inst.opr.dest, source, dest_mask, dest_repeat_offset);
 
+    if (source == spv::NoResult) {
+        LOG_ERROR("Source Not Loader!");
```

Comment:
fix this
also remove `!` from all messages and add a space
and you forgot closing `)` in some of them

Reply:
fix this ? what is this ?

Reply:
all is done